modules/layers/src/scatterplot-layer/scatterplot-layer.js

```diff
@@ -30,18 +30,58 @@ const DEFAULT_COLOR = [0, 0, 0, 255];
 
 const defaultProps = {
   radiusScale: {type: 'number', min: 0, value: 1},
-  radiusMinPixels: {type: 'number', min: 0, value: 0}, //  min point radius in pixels
-  radiusMaxPixels: {type: 'number', min: 0, value: Number.MAX_SAFE_INTEGER}, // max point radius in pixels
-  strokeWidth: {type: 'number', min: 0, value: 1},
-  outline: false,
+  radiusMinPixels: {type: 'number', min: 1, value: 1}, //  min point radius in pixels
```

Comment:
Please test the GeoJsonLayer in layer browser.

Reply:
Why is `min` changed to 1?

Reply:
because we need to avoid the "nothing to render" issue. The value should be at least 1.

Reply:
Users can set it to 0 if they want. We only want to change the default to 1.